Applications/spire/source/security_input_ui_tester/security_input_tester.cpp

```diff
@@ -50,8 +51,9 @@ void SecurityInputTester::showEvent(QShowEvent* event) {
           positioned = true;
         }
         if(m_dialog->exec() == QDialog::Accepted) {
-          addItem(QString::fromStdString(
-            Nexus::ToString(m_dialog->get_security())));
+          auto item_delegate = new CustomVariantItemDelegate(m_dialog);
```

Comment:
This is a memory leak, use automatic storage.

Reply:
Even with the dialog as the parent?